### PR TITLE
Added example to Scopes.md

### PR DESCRIPTION
**Pull request by dkrito**
Added an example on how to unset a user at logout.

### CODE REVIEW

**Comment by dkrito** [comment]
Trying this PR again

File: src/collections/_documentation/enriching-error-data/scopes.md [hunk 1]
```diff
@@ -60,6 +60,11 @@ add custom tags or to inform sentry about the currently authenticated user.
 
 {% include components/platform_content.html content_dir='configure-scope' %}
 
+This can also be applied when unsetting a user at logout:
+
+```javascript
+Sentry.configureScope(scope => scope.setUser({}));
```
Comment (MimiDumpling) [inline src/collections/_documentation/enriching-error-data/scopes.md]:
```suggestion
Sentry.configureScope(scope => scope.setUser(null));
```

**Comment by MimiDumpling** [comment]
@kamilogorek Denisse is gonna fix the dropdown before deployment. Just a heads up.

File: src/collections/_documentation/enriching-error-data/scopes.md [hunk 2]
```diff
@@ -60,6 +60,11 @@ add custom tags or to inform sentry about the currently authenticated user.
 
 {% include components/platform_content.html content_dir='configure-scope' %}
 
+This can also be applied when unsetting a user at logout:
```
Comment (MimiDumpling) [inline src/collections/_documentation/enriching-error-data/scopes.md]:
```suggestion
```

File: src/collections/_documentation/enriching-error-data/scopes.md [hunk 3]
```diff
@@ -60,6 +60,11 @@ add custom tags or to inform sentry about the currently authenticated user.
...
+
+```javascript
```
Comment (MimiDumpling) [inline src/collections/_documentation/enriching-error-data/scopes.md]:
```suggestion
```javascript
```

File: src/collections/_documentation/enriching-error-data/scopes.md [hunk 4]
```diff
@@ -60,6 +60,11 @@ add custom tags or to inform sentry about the currently authenticated user.
...
+Sentry.configureScope(scope => scope.setUser(null));
```
Comment (MimiDumpling) [inline src/collections/_documentation/enriching-error-data/scopes.md]:
```suggestion
```

File: src/collections/_documentation/enriching-error-data/scopes.md [hunk 5]
```diff
@@ -60,6 +60,11 @@ add custom tags or to inform sentry about the currently authenticated user.
...
+```
```
Comment (MimiDumpling) [inline src/collections/_documentation/enriching-error-data/scopes.md]:
```suggestion
```

**Comment by dkrito** [comment]
@kamilogorek @HazAT Hey 😃 This is my first time doing an include. Did I do it right?

File: src/collections/_documentation/enriching-error-data/unset-user/javascript.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+```javascript
+Sentry.configureScope(scope => scope.setUser(null));
```
Comment (HazAT) [inline src/collections/_documentation/enriching-error-data/unset-user/javascript.md]:
```suggestion
Sentry.setUser(null);
```

**Comment by HazAT** [comment]
Yes, include is correct, the code can be easier :)

**Comment by kamilogorek** [comment]
It could, but it lives in enriching-error-data section, which explains how `configureScope` works 😅

**Comment by MimiDumpling** [comment]
> It could, but it lives in enriching-error-data section, which explains how `configureScope` works 😅
> 
> https://docs.sentry.io/enriching-error-data/scopes/?platform=javascript#configuring-the-scope

@HazAT Since this section is about scopes, maybe it's better to keep `configureScope` in the code snippet? Also, happy to start a conversation about updating our other code snippets to highlight the "easier" code. 😸

**Comment by HazAT** [comment]
Ignore me, all good :shipit:

**Pull request merged**